cmake/DLAF_AddTest.cmake

```diff
@@ -86,10 +118,14 @@ function(DLAF_addTest test_target_name)
       message(FATAL_ERROR "Wrong MPIRANKS number ${DLAF_AT_MPIRANKS}")
     endif()
 
-    if (DLAF_AT_MPIRANKS GREATER MPIEXEC_MAX_NUMPROCS)
-      message(FATAL_ERROR "Impossible to have more than ${MPIEXEC_MAX_NUMPROCS}")
+    if (DLAF_AT_MPIRANKS GREATER MPIEXEC_MAX_NUMPROCS AND NOT DLAF_AT_MPI_OVERSUBSCRIBE)
```

Comment:
`max_numprocs` can be set to any number. How can you know if you need `OVERSUBSCRIBE`?